### ISSUE TITLE
Remove `plugin-auth` dependency for `plugin-dev`

### ISSUE THREAD

**Issue by ngmisl** [Wulf/create-rust-app#52]
```
❯ cargo fullstack
    Updating crates.io index
error: failed to select a version for `create-rust-app`.
    ... required by package `my-todo-app v0.1.0 (/home/teresa/dev/rust/my-todo-app)`
versions that meet the requirements `^8.0.0` are: 8.0.0

the package `my-todo-app` depends on `create-rust-app`, with features: `database_sqlite` but `create-rust-app` does not have these features.


failed to select a version for `create-rust-app` which could resolve this conflict
```

here's the cargo.toml:

[[bin]]
name = "fullstack"
path = ".cargo/bin/fullstack.rs"

[[bin]]
name = "tsync"
path = ".cargo/bin/tsync.rs"

[[bin]]
name = "my-todo-app"
path = "backend/main.rs"

[dependencies]
actix-files = "0.6.0"
actix-http = "3.0.0"
actix-multipart = "0.4.0"
actix-web = "4.0.1"
futures-util = "0.3.21"
create-rust-app = {version="8.0.0", features=["plugin_dev", "plugin_storage", "database_sqlite", "backend_actix-web"]}
serde_json = "1.0.79"
tsync = "1.2.1"

[dependencies.chrono]
features = ["serde"]
version = "0.4.19"

[dependencies.diesel]
default-features = false
features = ["sqlite", "r2d2", "chrono"]
version = "2.0.0-rc.1"

[dependencies.serde]
features = ["derive"]
version = "1.0.133"

[dependencies.tokio]
features = ["full"]
version = "1"

[package]
default-run = "my-todo-app"
edition = "2021"
name = "my-todo-app"
publish = false
version = "0.1.0"
[profile.dev]
debug-assertions = true

**Comment by Wulf** [comment]
hey @ngmisl thanks for reporting this and sorry you ran into it -- try the latest version and see if it fixes your issue!


```
cargo install create-rust-app_cli
```

**Comment by ngmisl** [comment]
gettng like 4 screens worth of errors now :(

**Comment by Wulf** [comment]
hey @ngmisl, I just realized I forgot to reply to you. I looked into your setup and reproduced the errors too haha. The problem is that the `dev` plugin needs the `auth` plugin at the moment. This shouldn't be the case but I need to remove this requirement. Again, sorry for replying so late, I think if you remove the dev plugin, you'll be fine.

Definitely need to setup `cargo-hack` so we can test different feature flag combinations
